app/serializers/appointment_serializer.rb

```diff
@@ -1,6 +1,37 @@
 class AppointmentSerializer < ActiveModel::Serializer
   attributes :id, :start_time, :status
 
+  CANCELED = :CANCELED
+  PAST = :PAST
+  PRESENT = :PRESENT
+  PENDING = :PENDING
```

Comment:
Since we send this data to front it would be easier to process it further when delivered lowercased

```suggestion
  CANCELED = :canceled
  PAST = :past
  PRESENT = :present
  PENDING = :pending
```